core/modules/wdb/QueryRunner.cc

```diff
@@ -340,23 +342,35 @@ class ChunkResourceRequest {
 
     ChunkResource getResourceFragment(int i) {
         proto::TaskMsg_Fragment const& fragment(_msg.fragment(i));
+        LOGS(_log, LOG_LVL_DEBUG, "fragment i=" << i);
         if (!fragment.has_subchunks()) {
-            StringVector tables;
+            DbTableSet dbTbls;
             for (auto const& scanTbl : _msg.scantable()) {
-                tables.push_back(scanTbl.db() + "." + scanTbl.table());
+                DbTable dbT(scanTbl.db(), scanTbl.table());
+                dbTbls.emplace(dbT);
```

Comment:
I think you can do `dbTbls.emplace(scanTbl.db(), scanTbl.table())`, saves a line of code and is probably more efficient.